### PR TITLE
Add Amazon Music

### LINKED ISSUE #3163: Amazon Music
So Amazon Music is very confusing branding wise, just read this site and try and workout what the name of the service is.

[Amazon Music sub-brands](https://www.amazon.com/gp/help/customer/display.html?nodeId=GW3PHAUCZM8L7W9L)

I'm happy with the below image but I don't know what branding link to use.

amazonmusic

**Name:** Amazon Music
**Website:** https://music.amazon.com/
**Official resources for icon and color:** https://press.aboutamazon.com/images-videos

### PR DESCRIPTION
**Pull request by adamrusted**
amazonmusic

**Issue:** closes #3163

**Similarweb rank:** N/A (Amazon Brand)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG pulled from homepage (behind login).
Hex is used extensively as the CTA color on the site.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -661,6 +661,11 @@
             "hex": "9146FF",
             "source": "https://luna.amazon.com"
         },
+        {
+            "title": "Amazon Music",
+            "hex": "25D1DA",
+            "source": "https://music.amazon.com"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:667]:
```suggestion
            "source": "https://music.amazon.com",
            "guidelines": "https://artists.amazonmusic.com/tools"
```
We should add the guidelines entry

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
The icon is a perfect match @adamrusted!
Let us add the guidelines too :)

**Comment by adamrusted** [comment]
> Should we use the indigo color instead `#4512ff`? Perhaps stack the word mark too as shown in guidelines?

I used the brand colour available when visiting Amazon Music - and the wordmark used there too. A lot of our Amazon brands use the horizontal wordmark, so I don't think we need to worry there necessarily.

**Comment by PratyushJayachandran** [comment]
> > Should we use the indigo color instead `#4512ff`? Perhaps stack the word mark too as shown in guidelines?
> 
> I used the brand colour available when visiting Amazon Music - and the wordmark used there too. A lot of our Amazon brands use the horizontal wordmark, so I don't think we need to worry there necessarily.

Okay then, should we add the guidelines entry to JSON?

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
It seems like the linter has started to fail even though the change was unrelated ti failure. @adamrusted can you re-do it?

File: icons/amazonmusic.svg
Comment (PratyushJayachandran) [inline icons/amazonmusic.svg:1]: When compared with their PNG I see [some differences in positioning.](https://github.com/simple-icons/simple-icons/assets/35000807/4e0d8ec3-0684-4f84-8a4b-4bce1dde3cf2)

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Pull request closed without merging**